lib/libzrepl/data_conn.c

```diff
@@ -591,19 +599,38 @@ uzfs_zvol_timer_thread(void)
 	zvol_info_t *zinfo;
 	time_t min_interval;
 	time_t now, next_check;
+	struct singly_node_list n_zvol_list, n_free_list;
```

Comment:
lets just make it as zvol_node_list, free_node_list kind of names

Reply:
ok